wrench/Cargo.toml

```diff
@@ -7,7 +7,7 @@ license = "MPL-2.0"
 
 [dependencies]
 base64 = "0.3"
-bincode = "1.0.0-alpha2"
+bincode = "0.8"
```

Comment:
Why from a 1.0 alpha to 0.8? Can someone explain to me what's going on with bincode's versioning? It's so confusing.